exporters/committime/collector_base.py

```diff
@@ -251,6 +279,85 @@ def get_repo_from_jenkins(self, jenkins_builds):
                 )
         # If no repo is found, we will return None, which will be handled later on
 
+    def get_metric_from_pipelineruns(self, pipelines, namespace):
+        """Expects a sorted array of build data sorted by app label"""
+        metrics = []
+        for pipeline in pipelines:
+            runs = pipelines[pipeline]
+
+            for run in runs:
+                repo_url = 'http://default'
+                commit_sha = 'default'
+                image_hash = 'default'
+                image_location = 'http://default'
+                try:
+                    # metric = self.get_metric_from_build(build, app, namespace, repo_url)
+                    metric = CommitMetric(pipeline)
+                    # By convention need to get image location from PARAMS
+                    params = run.status.pipelineSpec.params
+                    if params:
+                        for param in params:
+                            if param.name == "IMAGE_NAME":
+                                image_location = param.default
+                    # save each taskRun inside the pipeline
+                    task_runs = run.status.taskRuns
+                    if task_runs:
+                        for taskrun in task_runs:
```

Comment:
@jlozano-rh This will set repo_url and the others to the last task_run in the list. Is that intentional? I'm not really sure how it's structured for your use case.

Reply:
@KevinMGranger yes, it's intentional... I don't know any other way to walk through the `run` object to get those YAML properties. I mean, a `PipelineRun` has a list of executed tasks (a.k.a. `task_runs`), and in one of them we can find the **URL** and the **hash** of the last commit of the application. I hope I have answered your questions.

Reply:
Wouldn't those parameters be on the `PipelineRun`'s parameters instead?

Reply:
@KevinMGranger yes, that's right. I called `run` to each `PipelineRun` for simplicity.